firmware/arduino/chessboard/uart.ino

```diff
@@ -3,6 +3,10 @@ char buffer[6];
 int byteNum = -1; // -1 indicates that the start code hasn't been received
 char currentState = '0';
 char errorCode;
+volatile unsigned long previous_activation_time = 0;
+const unsigned long DEBOUNCE_TIME = 100; // Milliseconds
```

Comment:
You could save memory by using a preprocessor in the main code

Reply:
Agree with making DEBOUNCE a define just for consistency with other constants